cablelabs/transparent-security
# Snaps hcp ae tunnel

#### What does this PR do?
Fixes #208 
Creates a new VM using the snaps-hcp AMI, creates a GRE tunnel between the mininet host's clone egress port and the snaps-hcp machine for receiving telemetry report packets.
#### Do you have any concerns with this PR?
no
#### How can the reviewer verify this PR?
Run the lab_trial scenario
After that completes, manually run the following playbook on the mininet host:
ansible-playbook -i transparent-security.ini ./transparent-security/playbooks/scenarios/lab_trial/data-inspection.yml --extra-vars="scenario_send_protocol=UDP scenario_send_ip_version=4"

note: the supported send protocols are UDP & TCP and IP versions are 4 & 6 for a total of 4 different scenarios to check out.

#### Any background context you want to provide?
For the lab trial, we are planning on using snaps-hcp for our analytics engine and this is the initial work necessary to start on that work.
#### Screenshots or logs (if appropriate)
#### Questions:
- Have you connected this PR to the issue it resolves? yes
- Does the documentation need an update? not really quite yet
- Does this add new dependencies? not really
- Have you added unit or functional tests for this PR? not yet
